app/assets/javascripts/hated-foods.js

```diff
@@ -18,8 +16,43 @@ $(document).ready(function(){
     });
   };
 
-   $('.hated-food').children().on('click', function(){
+  $('.hated-food').find('button').on('click', function(){
     $(this).toggleClass('btn-danger');
   });
 
+  $("#favorite-number").bind('click', function(){
+    $("#favorite-number-selection").toggleClass('hidden')
+  });
+
+  $("#specific-food").on("ajax:success", function(e, data, status, xhr) {
+      var button = $('<button type="button" class="btn btn-danger one-banned-food" style="margin: 2px" data-id=' + data.id + '>'+data.name+'</button>');
+      $("#banned-food-list").append(button);
+      return $("input").val("")
+    }).on("ajax:error", function(e, xhr, status, error) {
```

Comment:
Mixed double and single quotes.<br>'error' is defined but never used.<br>'status' is defined but never used.<br>'xhr' is defined but never used.<br>'e' is defined but never used.